ui/legacy/src/components/cstudio-forms/data-sources/dropTargets.js

```diff
@@ -327,7 +335,7 @@
 
     _openContentTypeForm(contentType, type, control) {
       const self = this;
-      const path = `${self.baseRepoPath}/${contentType.replace(/\//g, '_').substr(1)}`;
+      const path = `${self.baseRepoPath}/${contentType.split('/').pop()}`;
```

Comment:
I don't think this works if someone decides to name a content type `/component/foo/bar`
```suggestion
      const path = `${self.baseRepoPath}/${contentType.replace(/^\/component(s?)\//, '')}`;
```